docs/sql-reference/00-sql-reference/20-system-tables/system-virtual-columns.md

```diff
@@ -0,0 +1,21 @@
+---
+title: system.virtual_columns
+---
+
```

Comment:
This diagram showing the virtual columns works very well, thanks for your help! @soyeric128